tests/jest/db/JobDAO.spec.js

```diff
@@ -0,0 +1,41 @@
+import PgBoss from 'pg-boss';
```

Comment:
did we ever figure out why this test hangs?  we'll need to resolve that before next deployment...